WFCORE-4226: Windows service recovery actions

mabrarov opened the pull request:
Turning on failure actions flag for Windows service. Propagation of exit code from launch scripts (Windows batch) of WildFly (for standalone and domain profile). ISSERVICE flag as start parameter of Windows service to transform exit code for Windows SCM (to not interleave with Windows standard error codes / messages).

Refer to [WFLY-10362](https://issues.jboss.org/browse/WFLY-10362) for details.

jamezp commented:
The commit message should be changed to `WFCORE-4226` not the moved WFLY JIRA.

mabrarov commented:
@jamezp,

> The commit message should be changed to `WFCORE-4226` not the moved WFLY JIRA.

If I'm not wrong - there is no way to change commit message in opened pull request without:

1. creation of new branch (and rebasing commits on to that new branch with changing of commit message)
1. opening new pull request

is it really needed for this case?

jamezp commented:
Sorry I didn't look at the commit messages. Generally speaking we use something like:
> [WFCORE-4226] This is the description of my commit

That said these commits should likely be squashed into a single commit as they seem to presenting a single concept.

mabrarov commented:
@jamezp ,

Is it acceptable if I create new pull request with one commit and fixed commit message and then will close (decline) this pull request with reference to the new one?

Should I name my source branch like feature/WFCORE-4226_exit_code_propagation?

mabrarov commented:
@jamezp,
 
> The commit message should be changed to `WFCORE-4226` not the moved WFLY JIRA.

Squashed all commits into a single one and changed commit message - removed intermediate fixes and added `[WFCORE-4226]`.

mabrarov commented:
This pull request is stale (it was waiting for review for ~2 years) and I do not plan to support it anymore. Closing / declining.
